src/style.css

```diff
@@ -0,0 +1,122 @@
+.page {
+  font-family: "Roboto", sans-serif;
+}
+
+.page__product-cards {
+  margin: 0;
+}
+
+.card {
```

Comment:
[Checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md) 10th item. You should create separate file for each BEM-block